s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html

```diff
@@ -40,10 +40,6 @@
                 <span i18n>Instrument:</span>
                 <strong>{{result.instrument}}</strong>
               </li>
-              <li>
-                <span i18n>Size:</span>
-                <strong>{{result.size}}</strong>
-              </li>
```

Comment:
Not only should size be removed, but `spacecraft` and `processing_level` metadata should be shown dynamically here